bin/hof-transpiler

```diff
@@ -1,53 +1,39 @@
 #!/usr/bin/env node
 
-var mkdirp = require("mkdirp");
+'use strict';
+
+const parsed = require('../lib/parse-args')(process.argv.slice(2));
+let sources = parsed.sources;
+let shared = parsed.shared;
+
+var mkdirp = require('mkdirp');
```

Comment:
looks like theres a mix of es5 and es6 - lets leave it all as vars until we es6ify or nuke it